django/template/base.py

```diff
@@ -86,7 +86,8 @@
 # Match BLOCK_TAG_*, VARIABLE_TAG_*, and COMMENT_TAG_* tags and capture the
 # entire tag, including start/end delimiters. Using re.compile() is faster
 # than instantiating SimpleLazyObject with _lazy_re_compile().
-tag_re = re.compile(r"({%.*?%}|{{.*?}}|{#.*?#})")
+tag_re = re.compile(r"({%.*?%}|{{.*?}}|{#.*?#})", re.DOTALL)
+tag_re_legacy = re.compile(r"({%.*?%}|{{.*?}}|{#.*?#})")
```

Comment:
Maybe like this to prevent repeating and clear for it's actually same.

```
tag = r"({%.*?%}|{{.*?}}|{#.*?#})"
tag_re = re.compile(tag, re.DOTALL)
tag_re_legacy = re.compile(tag)
```